uat/custom-components/src/test/java/com/aws/greengrass/LogGeneratorTest.java

```diff
@@ -17,19 +16,18 @@ public class LogGeneratorTest {
     @TempDir
     static Path tempPath;
     String logFileName = "localtest";
-    String logFileExtention = "log";
-    String logWriteFreqSeconds = "0.1";
+    String logWriteFreqMs = "100";
     String totalLogNumbers = "50";
     String fileSizeBytes = "1024";
     String fileSizeUnit = "KB";
     String componentName = "com.aws.greengrass.artifacts.LogGenerator";
-    String activeFileName = logFileName + "." + logFileExtention;
+    String activeFileName = logFileName + ".log";
```

Comment:
Why are we moving from a variable to a hard coded string?

Reply:
This is a test, variable wasn't a variable but a constant.